docs/UserGuide.md

```diff
@@ -211,25 +211,25 @@ Adds student details to the person list.
 
 Format: `add id/STUDENT_ID n/NAME e/EMAIL g/GRADE [gp/GROUP_NAME]​...`
```

Comment:
Is there a specific format for GROUP_NAME that the user should adhere to

Reply:
I don't think so. Any alphanumeric characters will do

Reply:
Thanks for pointing it out